doc/source/terrain.md

```diff
@@ -42,169 +178,212 @@ The aspect describes the orientation of strongest slope.
+This mode of shading the slopes often generates a map that is much more easily interpreted than the slope.
+
+The hillshade $hs$ is directly based on the slope $\alpha$ and aspect $\theta$, and thus also varies between the method of [Horn (1981)](http://dx.doi.org/10.1109/PROC.1981.11918) (default) and that of [Zevenbergen and Thorne (1987)](http://dx.doi.org/10.1002/esp.3290120107), and
+is often scaled between 1 and 255:
 
-As the hillshade is directly based on the slope and aspect, it varies between the method of [Horn (1981)](http://dx.doi.org/10.1109/PROC.1981.11918) (default) and that of [Zevenbergen and Thorne (1987)](http://dx.doi.org/10.1002/esp.3290120107).
+$$
+hs = 1 + 254 \left[ sin(alt) cos(\alpha) + cos(alt) sin(\alpha) sin(2\pi - azim - \theta) \right],
+$$
+
+where $alt$ is the shading altitude (90° = from above) and $azim$ is the shading azimuth (0° = north).
 
 Note, however, that the hillshade is not a shadow map; no occlusion is taken into account so it does not represent "true" shading.
 It therefore has little analytic purpose, but it still constitutes a great visualization tool.
 
-```{image} basic_examples/images/sphx_glr_plot_terrain_attributes_003.png
-:width: 600
-```
-
-```{eval-rst}
-.. minigallery:: xdem.terrain.hillshade
-        :add-heading:
+```{code-cell} ipython3
+hillshade = dem.hillshade()
+hillshade.plot(cmap="Greys_r", cbar_title="Hillshade")
 ```
 
+(curv)=
 ## Curvature
 
 {func}`xdem.DEM.curvature`
 
-The curvature map is the second derivative of elevation, which highlights the convexity or concavity of the terrain.
+The curvature is the second derivative of elevation, which highlights the convexity or concavity of the terrain.
 If a surface is convex (like a mountain peak), it will have positive curvature.
 If a surface is concave (like a through or a valley bottom), it will have negative curvature.
 The curvature values in units of m{sup}`-1` are quite small, so they are by convention multiplied by 100.
 
-The curvature is based on the method of [Zevenbergen and Thorne (1987)](http://dx.doi.org/10.1002/esp.3290120107).
+The curvature $c$ is based on the method of [Zevenbergen and Thorne (1987)](http://dx.doi.org/10.1002/esp.3290120107).
 
-```{image} basic_examples/images/sphx_glr_plot_terrain_attributes_004.png
-:width: 600
-```
+$$
 
-```{eval-rst}
-.. minigallery:: xdem.terrain.curvature
-        :add-heading:
+c = - 100 \frac{(h_{+0} + h_{-0} + h_{0+} + h_{0-}) - 4 h_{00}}{\Delta x \Delta y}.
+
+$$
+
+```{code-cell} ipython3
+curvature = dem.curvature()
+curvature.plot(cmap="RdGy_r", cbar_title="Curvature (100 / m)", vmin=-1, vmax=1, interpolation="antialiased")
 ```
 
+(plancurv)=
 ## Planform curvature
 
 {func}`xdem.DEM.planform_curvature`
 
-The planform curvature is the curvature perpendicular to the direction of slope, reported in 100 m{sup}`-1`.
+The planform curvature is the curvature perpendicular to the direction of slope, reported in 100 m{sup}`-1`, also based
```

Comment:
I had to check because I find the names counter-intuitive. To me, profile would mean perpendicular to the slope. I guess I am biased by the fact we often make cross profiles in glaciology 😉

Reply:
Double-checked now too!